### DISCUSSION TITLE
Can the installer be ran multiple times?

### DISCUSSION THREAD

**Discussion by Dr-Ari-Gami** [modoboa/modoboa#2733] [Installer]
If I want to change something in the installer.cfg, can I run it again, or will that damage my instance?

Thanks!

**Comment by Spitfireap** [comment]
Hi, 
If you want to change installer configuration, please reset your system before running it again. You can use the backup/restore feature of the installer : Backup, transfer backup, reset system, import backup, edit your installer.cfg file inside your backup, run installer again with restore argument.

**Reply by Dr-Ari-Gami** [reply]
I have other services installed on the same server. Is there a way to just delete everything associated with modoboa and do a clean install without resetting my whole server?

Thanks!
